### PR TITLE
[sailjail] Remove plugin dir and conf path options. Fixes JB#53041

### PR DESCRIPTION
**Pull request by Tomin1**

### CODE REVIEW

**Comment by izh1979** [comment]
Should we remove the profile argument too?

**Comment by izh1979** [comment]
I.e. wouldn't it be possible to ask sailjail to run some applications with more permossions than it is requested in its *.desktop?

**Comment by Tomin1** [comment]
> Should we remove the profile argument too?

No, that's actually necessary to define the desktop file to use. It's not about firejail profiles but sailjail profiles which are usually .desktop files.

> I.e. wouldn't it be possible to ask sailjail to run some applications with more permossions than it is requested in its *.desktop?

That's not what --profile is about.

**Comment by izh1979** [comment]
Then LGTM.

File: src/sailjail.c [hunk 1]
```diff
@@ -384,36 +370,22 @@ int main(int argc, char* argv[])
...
+        /* The config file may be (and usually is) missing */
+        if (g_file_test(DEFAULT_CONF_FILE, G_FILE_TEST_EXISTS)) {
+            jail_conf_load(conf, DEFAULT_CONF_FILE, NULL);
         }
 
-        if (ok) {
-            /* Allocate new parser */
-            g_option_context_free(options);
+        /* Allocate new parser */
+        g_option_context_free(options);
```
Comment (monich) [inline src/sailjail.c:379]: If config file is no longer configurable from the the command line, there's no need to parse the command line twice anymore. Just load defaults from the config file and parse the command line. Once.

Comment (Tomin1) [reply src/sailjail.c]: Thanks. Cleaned up that unnecessary code.

**Review by spiiroin — APPROVED** [review]
LGTM

**Pull request merged**